src/lib/stitching/exchange/v2/stitching.ts

```diff
@@ -134,6 +134,73 @@ export const exchangeStitchingEnvironment = ({
     },
   }
 
+  const inquiryOrderResolvers = {
+    isInquiryOrder: {
+      fragment: gql`
+        fragment CommerceOrderIsInquiryOrder on CommerceOfferOrder {
+          impulseConversationId
+        }
+      `,
+      resolve: async (order) => {
+        const { impulseConversationId } = order
+        return Boolean(impulseConversationId)
+      },
+    },
+    conversation: {
+      fragment: gql`
+        fragment CommerceOrderConversation on CommerceOfferOrder {
+          impulseConversationId
+        }
+      `,
+      resolve: async (order, _args, context, info) => {
+        const { impulseConversationId } = order
+        if (!impulseConversationId) return null
+
+        return info.mergeInfo.delegateToSchema({
```

Comment:
Aaaah, I just noticed we don't have root-level conversation query so the only way to get it is from `me`. This works well for the collector side but I am wondering if it works in volt where the logged-in user is not the owner of the conversation?

Can we get the `conversationLoader` from `context` and use that for getting the conversation instead of `deligateToSchema` (which in the end calls the same loader because impulse is REST) and leave authorization for impulse?